domains/client/domain-executor/README.md

```diff
@@ -19,3 +36,54 @@ Each executor instance, whether for system domain or core domain, consists of th
   - Executor components specfic to the system domain.
 - `core_{block_processor,bundle_producer,bundle_proposer,worker}.rs`/`core_gossip_message_validator`
   - Executor components specfic to the core domain.
+
+## Run a local testnet
+
+Clone the repo and build the executables:
+
+```bash
+git clone https://github.com/subspace/subspace
+cd subspace
+cargo build --release --bin subspace-farmer --bin subspace-node
+```
+
+Start a local testnet:
+
+```bash
+# Run a primary chain, a system domain and a core domain node in one command:
+subspace-node --dev -d tmp --ws-port 4567 -- --alice --dev --ws-port 5678 -- --domain-id 1 --alice --dev --ws-port 6789
+
+# Prepare the reward address beforehand and start a farmer in another terminal:
+subspace-farmer --base-path tmp-farmer farm --plot-size 100M --reward-address [ADDRESS] --node-rpc-url ws://127.0.0.1:4567
+```
+
+```
+2023-01-01 09:46:37.818  INFO tokio-runtime-worker runtime::domains: [CoreDomain] Submitted bundle
```

Comment:
do we need the logs here ?
Seems a lot of noise

Reply:
It wouldn't print these logs though. Looks like you had `RUST_LOG` used. It is a bit tidier without debug messages by default.